.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,74 @@
+name: Publish Bailo Package to PyPI
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    name: Build distribution
+    runs-on: ubuntu-latest
+    outputs:
+      files-changed-output: ${{ steps.files-changed.outputs.any_changed }}  
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      # Check for changed init file (for new version)
+      - name: Get changed init files
+        id: files-changed
+        uses: tj-actions/changed-files@v40
+        with:
+          files: |
+            lib/python-beta/src/bailo/__init__.py
```

Comment:
I just want to point out, this will try and publish to PyPi if it detects any change in this file (Not just the version number).  However, I'm satisfied that it wouldn't be an issue with correct versioning.